charts/merlin/ci/ci-values.yaml

```diff
@@ -1,6 +1,7 @@
 global:
   oauthClientID: "global-client-123"
   mlp:
+    fullnameOverride: "mlp"
```

Comment:
Curious - why is this added ?

Reply:
It wasn't added in the `charts/merlin/values.yaml` until this PR , and i didn't realized that the same override was added on ci-values. I have removed this from the ci-values.

Reply:
This is so that the MLP url is deterministic. I remember that MLP url is also part of the Merlin configuration. Without the override, there's no guarantee that the default MLP url is correct, when you install Merlin + MLP using the default value.

Reply:
Ahh ok.. Got it. I guess Merlin -> MLP calls work as is, but probably not for other things like VS routes.